packages/clay-data-provider/stories/index.tsx

```diff
@@ -209,4 +209,42 @@ storiesOf('Components|ClayDataProvider', module)
 	))
 	.add('with variables change and storage', () => (
 		<ClayDataProviderWithVariablesAndStorage />
-	));
+	))
+	.add('toggle poll', () => {
+		const [poll, setPoll] = useState(false);
+
+		const {resource} = useResource({
+			link:
+				'https://api-public.sandbox.pro.coinbase.com/products/BTC-USD/trades',
+			pollInterval: poll ? 1000 : 0,
+			variables: {limit: 10},
+		});
+
+		// eslint-disable-next-line no-console
+		console.log(resource);
```

Comment:
is this log necessary for our storybook?

Reply:
This is only for viewing polling in action when it is turned off and on.